scripts/generatePiImage.sh

```diff
@@ -2,7 +2,7 @@
 # Not that it really matters, but it'll save us 50 megs or so
 NEW_JAR=$(realpath $(find . -name photonvision\*-linuxarm32.jar))
```

Comment:
is this still correct? Don't we need the 64-bit arm jar now?

Reply:
We will, once libcamera goes in. For now, arm32 is still technically correct